android-json-form-wizard/src/main/java/com/vijay/jsonwizard/widgets/RepeatingGroupFactory.java

```diff
@@ -305,23 +366,26 @@ private void addRequiredValidator(JSONObject jsonObject, MaterialEditText editTe
         }
     }
 
-    protected void addOnDoneAction(TextView textView) {
+    protected void addOnDoneAction(final TextView textView, final ImageButton doneButton, final WidgetArgs widgetArgs) {
```

Comment:
This fixes a bug that has to do with state management explained here https://github.com/OpenSRP/opensrp-client-native-form/pull/476/files/877163f20bacfba6fbfb6eeb77f053c9813ec04f#r484283652.

While this might break existing code, unfortunately, I don't see an easier way for this fix to be applied, since we need to move to passing state in the method to fix this bug.

Any ideas?